spec/article-list-model-spec.js

```diff
@@ -0,0 +1,19 @@
+"use strict";
+
+showOff("ArticleList", function(){
+  var articles = new ArticleList();
+  var articleObj = "article stub";
+
+  theThingy(".addArticle adds an article to the list, returnArticleByIndex returns that article stored in 0 position", function(){
```

Comment:
So, I'm deliberately not reviewing the test framework, but....
.
.
.
.
theThingy????